logging/overlays/nerc-ocp-obs/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../base
```

Comment:
Remember that when you configure a BackingStore like our `metrics-backing-store`, in the beginning we set it to `storage: 400Gi` and `numVolumes: 3`. Because we kept running out of log storage we had to keep increasing up to `numVolumes: 24`. At 24, we got a different error and through Red Hat Support Cases we figured out that 20 is the max `numVolumes` for a BackingStore, so we bumped it down to 20, [see the commits here to BackingStore](https://github.com/OCP-on-NERC/nerc-ocp-config/commits/main/cluster-scope/overlays/nerc-ocp-infra/metrics-bucket-storage/backingstore.yaml). So when choosing the number of volumes, keep in mind that the `numVolumes * storage == 10Ti` is what we are looking for, and it creates `numVolumes` PVCs in the openshift-storage namespace, [see here](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-storage/core~v1~PersistentVolumeClaim?name=metrics-backing-store). I think it's good to have more than one volume if we have multiple nodes in the cluster.

Reply:
Also the LokiStack configures `storageClassName: ocs-external-storagecluster-ceph-rbd`. We likely don't want to use the default storage class, but a separate one. I suggest we create a separate storage class for logging persistent volumes. We should [count the total capacity of all PVCs here in the `openshift-logging` namespace](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-logging/core~v1~PersistentVolumeClaim) and create a separate StorageClass for that as well, leaving some space to grow the `wal-logging-loki-ingester` PVCs which have run out of storage in the past, [see this issue Loki Logging wal-logging-loki-ingester PVCs reporting no space left on device](https://github.com/nerc-project/operations/issues/195).